src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -274,6 +277,11 @@ void SetParameter()
          Aux_Error( ERROR_INFO, "Incorrect parameter %s = %d !!\n", "CCSN_Is_PostBounce", CCSN_Is_PostBounce );
    }
 
+// CCSN_Min_AngRes must be larger than CCSN_Min_AngRes
+   if ( CCSN_Min_AngRes > 0.0  &&  CCSN_Max_AngRes > 0.0 )
+      if ( CCSN_Min_AngRes <= CCSN_Max_AngRes )
+         Aux_Error( ERROR_INFO, "%s = %d must be larger than %s = %d  !!\n", "CCSN_Min_AngRes", CCSN_Min_AngRes, "CCSN_Max_AngRes", CCSN_Max_AngRes );
```

Comment:
- combine the `if` statement
- the format `%d` is incorrect
- check `OPT__FLAG_REGION` is enabled when `CCSN_Min_AngRes` and/or `CCSN_Max_AngRes` are set.
- print warning message if `CCSN_Min_AngRes < 2.0 * CCSN_Max_AngRes` in which `CCSN_Max_AngRes` is not strictly followed.
- The code in my version is not optimized yet. I prefer to convert the unit of `CCSN_Min/Max_AngRes` to radian in this function in order to save time.

Reply:
- I combined the `if` statement
- I corrected the format
- I added checking `OPT__FLAG_REGION`
- I also added this warning message
- I converted `CCSN_Ang_Min/Max` to radian